unity_buy_android/unity_buy_plugin/build.gradle

```diff
@@ -0,0 +1,55 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 26
+    buildToolsVersion "26.0.0"
+
+    defaultConfig {
+        minSdkVersion 16
```

Comment:
I think moving  to 19 makes sense. I was only targeting 16 because that is what Unity exports by default by more than 90% of users are on >= 19 it looks like. 

Also, it looks like the MBSDK's pay module is set to a minimum of 19 which would be nice to use for the Unity Android Pay implementation.